File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/ios/Classes/Model/IDOStressCalibrationParamModel.swift
//
//  IDOStressCalibrationParamModel.swift
//  protocol_channel
//
//  Created by hc on 2023/10/12.
//

import Foundation
/// Set Stress Calibration Event Code
@objcMembers
public class IDOStressCalibrationParamModel: NSObject, IDOBaseModel {
    /// Stress score, ranging from 1 to 10
    public var stressScore: Int
    /// 0: Start calibration setting
    /// 1: Cancel calibration setting
    public var status: Int
    
    enum CodingKeys: String, CodingKey {
        case stressScore = "stress_score"
        case status = "status"
    }
    
    public init(stressScore: Int,status: Int) {
        self.stressScore = stressScore
        self.status = status
    }
    
    public func toJsonString() -> String? {
        if let jsonData = try? JSONEncoder().encode(self) {
            let jsonString = String(data: jsonData, encoding: .utf8)
            return jsonString
        }
        return nil
    }
}

File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/android/src/main/kotlin/com/idosmart/model/IDOStressCalibrationParamModel.kt
//
//  IDOStressCalibrationParamModel.kt
//  protocol_channel
//
//  Created by hedongyang on 2023/10/30.
//

package com.idosmart.model

import com.google.gson.GsonBuilder
import com.google.gson.annotations.SerializedName

///
/// Set Stress Calibration Event Code

open class IDOStressCalibrationParamModel(stressScore: Int, status: Int) : IDOBaseModel {

    /// Stress score, ranging from 1 to 10
    @SerializedName("stress_score")
    var stressScore: Int = stressScore

    /// 0: Start calibration setting
    /// 1: Cancel calibration setting
    @SerializedName("status")
    var status: Int = status


    

    override fun toJsonString(): String {
        return GsonBuilder().create().toJson(this).toString()
    }
}
    